miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/BoxRestController.java

```diff
@@ -561,8 +556,10 @@ public void setColumns(int columns) {
       // Build the diffs for this box
       List<DiffMessage> diffs = new ArrayList<>();
       Box box;
+      Map<String, BoxableView> boxables;
       try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)